lib/ansible/plugins/lookup/passwordstore.py

```diff
@@ -51,6 +51,14 @@
         type: bool
         default: 'no'
         version_added: 2.7
+      chars:
+        description:
+           - Define comma separated list of names that compose a custom character set in the generated passwords.
+           - 'By default generated passwords contain a random mix of upper and lowercase ASCII letters, the numbers 0-9 and punctuation (". , : - _").'
```

Comment:
```suggestion
           - 'By default generated passwords contain a random mix of upper and lowercase ASCII letters, the numbers 0-9 and these punctuation characters (". , : - _").'
```

To make sure these are the only characters added